# Case Study on Measures of Central Tendency and Dispersion

# An Institution wishes to find out their student’s ability in maths, reading andwriting skills. The Institution wants to do an exploratory study to check thefollowing information.

In [3]:
import pandas as pd
import numpy as np
import seaborn as sns

In [4]:
data = pd.read_excel(r'C:\Users\Home\Downloads\StudentsPerformance.xlsx' )

In [5]:
data

gender race/ethnicity parental level of education         lunch  \
0    female        group C            some high school  free/reduced   
1    female        group B                 high school  free/reduced   
2    female        group B            some high school  free/reduced   
3    female        group B                some college      standard   
4    female        group C                some college  free/reduced   
..      ...            ...                         ...           ...   
995    male        group E                some college      standard   
996    male        group A                some college      standard   
997    male        group D                some college      standard   
998    male        group E          associate's degree  free/reduced   
999    male        group E           bachelor's degree      standard   

    test preparation course  math score  reading score  writing score  
0                      none           0             17             10  
1                      none           8             24             23  
2                      none          18             32             28  
3                      none          11             38             32  
4                      none          22             39             33  
..                      ...         ...            ...            ...  
995               completed          99             87             81  
996               completed         100             96             86  
997               completed         100             97             99  
998               completed         100            100             93  
999               completed         100            100            100  

[1000 rows x 8 columns]

1. Find out how many males and females participated in the test

In [6]:
print(list(data['gender']).count('male'))

482


In [7]:
print(list(data['gender']).count('female'))

518


Number of male participants = 482
Number of female participants = 518

2. What do you think about the students' parental level of education?

In [15]:
print(list(data['parental level of education']).count('some college'))

226


In [16]:
print(list(data['parental level of education']).count('some high school'))

179


In [17]:
print(list(data['parental level of education']).count('high school'))

196


In [18]:
print("Maximum: ",data["math score"].max())


Maximum:  100


In [19]:
print("Minimum: ",data["math score"].min()) 

Minimum:  0


5. The management needs your help to give bonus points to the top 25% of students based on their maths score, so how will you help the management to achieve this.


In [ ]:
number = int(input('Enter the marks'))
if(number > 80):
 Print('Eligible for bonus points')
else:
 Print('Not eligible for bonus points')
 print('End')